microservice-transaction-sample/customer-service/src/main/java/sample/customer/model/Customer.java

```diff
@@ -30,8 +31,8 @@ public Customer(int id, String name, int creditLimit, int creditTotal) {
   public static void put(
```

Comment:
```suggestion
  public static void insert(
```
It might also be better to update the method name from `put` to `insert` so that we can avoid unnecessary confusion when users see this sample code.

However, if it takes a lot of time and effort, I think we can keep this method name as is for now.

What do you think?

Reply:
@kota2and3kan  Thanks for reviewing!
I've applied your suggestions in [789b3c2](https://github.com/scalar-labs/scalardb-samples/pull/74/commits/789b3c21aa09be5452f88338e8d655bb8ad27a54). PTAL!